packages/mobile/App.svelte

```diff
@@ -132,8 +132,9 @@
     html,
     body {
         @apply bg-white;
-        @apply select-none;
```

Comment:
The `@apply select-none;` was replaced by `user-select: none;` because when added `-webkit-user-select: none;` appear a warning indicating `user-select: none;` must exist together. Also in this case is more clear that we're targeting both browser engines: Webkit / Blink of the respective webviews.